cpp/scenario/gazebo/CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+#  This project is dual licensed under LGPL v2.1+ or Apache License.
+#
+# -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -
+#
+#  This software may be modified and distributed under the terms of the
+#  GNU Lesser General Public License v2.1 or any later version.
+#
+# -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# ================
+# Extra Components
+# ================
+
+set(EXTRA_COMPONENTS_HDRS
+#    ${CMAKE_CURRENT_SOURCE_DIR}/include/scenario/gazebo/components/ECMMutex.h
+#    ${CMAKE_CURRENT_SOURCE_DIR}/include/scenario/gazebo/components/PreUpdateConditionVariable.h
```

Comment:
This commented files are going to be added later?

Reply:
I removed these lines yesterday but the commit is still staging in my machine waiting for a rebase. I'm waiting for other comments to do a single rebase with all the edits :)

Reply:
Addressed in https://github.com/robotology/gym-ignition/pull/158/commits/115309bb6bc33a1812ab82b95bf46c9f0a754ba1